federation/scim/src/main/resources/META-INF/jboss-deployment-structure.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Looks like you have been using this extension but running the Wildfly-based releases of Keycloak?

We should not need it anymore as we are running on Quarkus now.